### DISCUSSION TITLE
Laravel Sail sebagai standar local development berbasis docker container

### DISCUSSION THREAD

**Discussion by andriawan** [buku-masjid/buku-masjid#16] [General]
Apakah memungkinkan untuk menerapkan ini mas @nafiesl? Saya pribadi sering memakai ini. Sebelumnya, saya sempat meracik dokcer sendiri untuk kebutuhan development di lokal atau cloud IDE. Sejak mengenal sail, saya pribadi memutuskan untuk selalu memakai Laravel Sail di setiap development berbasis docker container. Alasan utamanya ialah, Sail mendapatkan first class support dari tim pengembangan laravel.

Ini halaman officialnya https://laravel.com/docs/10.x/sail

**Comment by nafiesl** [comment]
Halo mas @andriawan, setelah membaca sekilas tentang sail, sepertinya bagus juga mas. Kalau mas @andriawan berkenan untuk buat PR, alhamdulillah.

Ada beberapa catatan:
1. Di deskripsi PR, kasih "how to test" nya, maksudnya perintah apa aja yang perlu dijalankan
2. Apakah memungkinkan jika mysql nya menggunakan mysql yang sudah ada di localhost?
3. Saat ini proyek ini pakai `yarn`, apakah memungkinkan di sail pakai yarn saja, tanpa npm?

Silakan mas :pray:

**Reply by andriawan** [reply]
sepertinya ini sudah tidak relevan karena saya lihat sudah ada konfigurasi developmen docker secara lokal pada codebase terbaru. Diskusi ini bisa ditutup saja mas @nafiesl

**Reply by nafiesl** [reply]
Siap mas @andriawan terima kasih mas.
